app/Support/PluginManager.php

```
@@ -1885,6 +2068,232 @@ public function loadActivePlugins(): void
+                $this->finalizePendingPluginUpdate($state);
+                @unlink($markerPath);
+            } catch (\Throwable $e) {
+                if (is_array($state)) {
+                    $this->rollbackPendingPluginUpdate($state, $e);
+                    @unlink($markerPath);
+                } else {
+                    SecureLogger::error('[PluginManager] Invalid pending plugin update marker', [
+                        'path' => $markerPath,
+                        'error' => $e->getMessage(),
+                    ]);
+                }
```

PHPMD (2.15.0)

[error] The method finalizePendingPluginUpdates uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)